validate/setup-guides/do/sgDroplet.md

```diff
@@ -115,18 +127,23 @@ In the **`ports`** column, we will add in the `start` port then the `end` port,
 
 We will **leave** the `All IPv4` and `All IPv6` items.
 
+##
 :::info
-Ports 9000 and 9001 are used for a Public (9000) and Peer-to-Peer (9001) **API** access.   Ports 9000 and 9001 are configurable and will be dependent on the Hypergraph or State Channel network you connect to.  Node Operators will need to learn what ports are opened for access to the State Channels, and update their firewall (change) accordingly
+Ports 9000 and 9001 are used for a Public (9000) and Peer-to-Peer (9001) **API** access.  Ports 9000 and 9001 are configurable and will be dependent on the access rules for the specific Hypergraph or Metagraph network you connect to.  **99%** of the tim, the ports used here can be any port the Node Operator wants to use.  Most Hypergraph and Metagraphs will not restrict the ports to be used.  However, Node Operators will need to learn what ports are opened for access to the Metagraphs, and update their firewall (change) accordingly.
 :::
 
-### MAINNET 2.0 LAUNCH REQUIREMENT
+## MAINNET 2.0 LAUNCH REQUIREMENT
```

Comment:
We're past this launch so I think this should be updated to something more descriptive of the current network state. It could also probably be merged with the section above since it's about opening ports.